src/main/java/magentoPage/DefaultAddressPage.java

```diff
@@ -0,0 +1,112 @@
+package magentoPage;
+
+import object.address.Address;
+import com.github.javafaker.Faker;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import utils.property.PropertyUtil;
+
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+
+public class DefaultAddressPage {
+    private WebDriver driver;
+    @FindBy(id = "telephone")
+    WebElement phoneInput;
+    @FindBy(id = "street_1")
+    WebElement streetInput;
+    @FindBy(id = "city")
+    WebElement cityInput;
+    @FindBy(id = "region_id")
+    WebElement regionSelect;
+    @FindBy(id = "zip")
+    WebElement zipCodeInput;
+    @FindBy(id = "country")
+    WebElement countrySelect;
+    @FindBy(xpath = "//button[contains(@class,'save')]")
+    WebElement saveAddressButton;
+    private final Properties testData = PropertyUtil.getProperties("testsData.properties");
```

Comment:
Need help explaining. I have a lot of property files that are categorized, if I put all the data in one general file then I think it might get confusing.

Reply:
Accepted, next time I will do one property file.